tool/loadgen/main.go

```diff
@@ -53,32 +53,45 @@ func main() {
 	if cAdvisorURL != "" && containers == "" {
 		panic("flag -containers is required when -cadvisor is provided")
 	}
-	if strings.Contains(containers, ",") {
-		panic("not implemented: only one container name supported")
-	}
-	containerName := containers
 
 	log.Printf("Target is %q", targetURL)
 
 	waitUntilReady(targetURL, maxWait)
 	warmUp(targetURL, rps, warmupDuration)
 
-	stats := Stats{}
+	statsMap := make(map[string]Stats)
```

Comment:
We could keep using the shorter name, it is uncommon in Go to name variables after their types.

```suggestion
	stats := make(map[string]Stats)
```